### DISCUSSION TITLE
I ran my MiXCR postanalysis individual program and received an error message with only a red "2"

### DISCUSSION THREAD

**Discussion by kankanjiuzou12138** [milaboratory/mixcr#1581] [Q&A] [answered]
Here is my program as follow
<img width="621" alt="截屏2024-03-11 12 03 53" src="https://github.com/milaboratory/mixcr/assets/162944666/d2543d1c-a12e-455d-b1c0-650882ad2299">
:
mixcr -Xmx14g postanalysis individual --default-downsampling cumtop-read --default-weight-function read --only-productive --drop-outliers /Volumes/WD_BLACK/F23A040007233_PEOhraiR/results_clns/sample01.clns /Post_result01.json

**Comment by mizraelson** [comment, marked as answer]
Hi, 
You should specify the percent for cumulative top, ex.:

`--default-downsampling cumtop-read-80`
